modules/python/src2/cv2.cpp

```diff
@@ -411,12 +411,84 @@ PyObject* pyopencv_from(const Mat& m)
     return o;
 }
 
-template<typename _Tp, int m, int n>
-PyObject* pyopencv_from(const Matx<_Tp, m, n>& matx)
-{
-    return pyopencv_from(Mat(matx));
+#define PYOBJ_FROM(_Tp) \
+static PyObject* pyopencv_from(const _Tp& src) \
+{ \
+    return pyopencv_from(Mat(src)); \
```

Comment:
Currently, the results are tuples, not even Numpy ararys. And as you indicated, it is inconsistent!